libcodechecker/logger.py

```diff
@@ -54,137 +40,126 @@ def debug_analyzer(self, msg, *args, **kwargs):
 
 logging.setLoggerClass(CCLogger)
 
-
-class CustomFormatter(logging.Formatter):
-    """
-    Custom formatter to print log level in case of ERROR, WARNING
-    or CRITICAL.
-    """
-
-    # Short format strings where only the time and the message is printed.
-    format_string_short = {
-        'info_fmt':  '[%(asctime)s] - %(message)s',
-        'error_fmt': '[%(levelname)s] [%(asctime)s] - %(message)s',
-        'debug_fmt': '[%(asctime)s] {%(name)s} [%(process)d] <%(thread)d> - '
-                     '%(filename)s:%(lineno)d %(funcName)s() - %(message)s'
+DEFAULT_LOG_CFG = os.path.join(os.path.dirname(__file__),
+                               'logger.conf')
+
+
+# Default config wich can be used if reading log config from a
+# file fails.
+DEFAULT_LOG_CONFIG = '''{
```

Comment:
DEFAULT_LOG_CFG and DEFAULT_LOG_CONFIG are almost the same names. Maybe it would be better to pick something that reflects the difference between the two. Also, do we need both? Can't we just fall back to the default config file if the user provided is not found? I am not sure if we would like to handle the cases gracefully when the files from the package are inaccessible for some reason.

Reply:
I've renamed it.
The minimal configuration it there as a backup so we can see log messages even if there is a configuration error in the log config file.